### DISCUSSION TITLE
可否先用Pyarmor加密项目中的N个脚本，再将这些脚本用Cython转换成pyd文件增加一层防护，最后再用pyinstall打包？

### DISCUSSION THREAD

**Discussion by mryiersan** [dashingsoft/pyarmor#1966] [General]
我完全是小白，所以可能这个问题比较菜，还请见谅。想请问：
1.Pyarmor加密后的脚本是否可以再被正常的转化成pyd脚本？
2.这样做的话是否会更安全？有没有必要这么做？

**Comment by chenpaner** [comment]
将这些脚本用Cython转换成pyd文件之后你再用Pyarmor调用不会报错么？我测试转pyd后会报错啊，好像时说Pyarmor许可证问题，
貌似错误提示说明，在先用 PyArmor 加密后再转换为 pyd 的过程中，授权检查机制失败了。可能的原因包括：
授权验证失败：PyArmor 会在加密代码中嵌入授权检查，转换为 pyd 后，验证机制可能无法正确识别合法授权。
你测试了能运行pyd的么？
